app/assets/translations/appConstants/bebbo/en.ts

```diff
@@ -391,7 +390,7 @@ const en = {
                 "dataSaveroff": "Do you want to switch off data saver mode?",
                 "dataSaveron": "Do you want to switch on data saver mode?",
                 "parentGender": "Gender",
-                "successOnboardingImport": "You have successfully uploaded your information.",
+                "successOnboardingImport": "You have successfully uploaded your  information.",
```

Comment:
Is this extra space intentional?

Reply:
Hello @Akhror , No this space is not intentional and we can update in string file. kindly create one task for all string changes.